doc/onboarding-extras.md

```diff
@@ -13,6 +13,7 @@
 | `lib/{crypto,tls,https}` | @nodejs/crypto |
 | `lib/domains` | @misterdjules |
 | `lib/fs`, `src/{fs|file}` | @nodejs/fs |
+| `lib/internal/url` | @nodejs/url |
```

Comment:
I don’t think there is anybody in particular to ping for `lib/url`, and I think leaving this as-is is going to be fine. If it turns out to be a horrible idea, we can add a note or find people to @mention for `lib/url`.